pkg/kubectl/cmd/create_pvc.go

```diff
@@ -0,0 +1,222 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/kubernetes/pkg/api"
+	internalversionstorage "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/core/internalversion"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/kubectl/util/i18n"
+)
+
+var (
+	pvcLong = templates.LongDesc(i18n.T(`
+		Create a persistent volume claim using specified subcommand`))
+
+	pvcExample = templates.Examples(i18n.T(`
+	    # Create a persistent volume claim named "myclaim" with "ReadWriteOnce" access mode and "1Gi" capacity.
+		kubectl create persistentvolumeclaim myclaim --access-mode=ReadWriteOnce --capacity=1Gi
+
+		# Create a persistent volume claim named "myclaim" with "ReadWriteOnce" access mode, "1Gi" capacity and a storage class named "HighQuality".
+		kubectl create persistentvolumeclaim myclaim --access-mode=ReadWriteOnce --capacity=1Gi --storage-class=HightQuality
+
+		# Create a persistent volume claim named "myclaim" with "ReadWriteOnce" access mode, "1Gi" capacity, requiring a persistent volume named "HighQualityVolume".
+		kubectl create persistentvolumeclaim myclaim --access-mode=ReadWriteOnce --capacity=1Gi --volume-name=HightQualityVolume
+
+		# Create a persistent volume claim named "myclaim" with multiple access modes and "1Gi" capacity.
+		kubectl create persistentvolumeclaim myclaim --access-mode=RWO,RWX --capacity=1Gi`))
+)
+
+type CreatePVCOptions struct {
+	Name         string
+	AccessModes  []api.PersistentVolumeAccessMode
+	Capacity     *resource.Quantity
+	StorageClass *string
+	VolumeName   string
+
+	DryRun       bool
+	OutputFormat string
+	Namespace    string
+	Client       internalversionstorage.PersistentVolumeClaimInterface
+	Mapper       meta.RESTMapper
+	Out          io.Writer
+	PrintObject  func(obj runtime.Object) error
+}
+
+// PersistentVolumeClaim is a command to ease creating PersistentVolumeClaims.
+func NewCmdCreatePersistentVolumeClaim(f cmdutil.Factory, cmdOut io.Writer) *cobra.Command {
```

Comment:
lint

Reply:
the name of the function?

Reply:
seems now comments must same with func name

Reply:
ok, I can change this, but seems for other create series commands, the comment of this function just uses the name of the resource.